Cargo.toml

```diff
@@ -81,9 +82,9 @@ usb = ["dep:stm32-usbd"]
 stm32g431 = ["stm32g4/stm32g431", "cat2"]
 stm32g441 = ["stm32g4/stm32g441", "cat2"]
 stm32g473 = ["stm32g4/stm32g473", "cat3", "adc3", "adc4", "adc5"]
-stm32g474 = ["stm32g4/stm32g474", "cat3", "adc3", "adc4", "adc5"]
+stm32g474 = ["stm32g4/stm32g474", "cat3", "adc3", "adc4", "adc5", "hrtim", "stm32-hrtim/stm32g474"]
 stm32g483 = ["stm32g4/stm32g483", "cat3", "adc3", "adc4", "adc5"]
-stm32g484 = ["stm32g4/stm32g484", "cat3", "adc3", "adc4", "adc5"]
+stm32g484 = ["stm32g4/stm32g484", "cat3", "adc3", "adc4", "adc5", "hrtim", "stm32-hrtim/stm32g484"]
```

Comment:
How is this best done? I want the hrtim feature to be optional (just like `can` and `cortic`) but only available for `stm32g474`, `stm32g484`.

Is there any other/better way than 

```rust
#[cfg(feature = "hrtim", not(any(feature = "stm32g474", feature = "stm32g484"))]
compile_error!("`hrtim` is only available for stm32g474 and stm32g484");
```